finality-aleph/src/party/member.rs

```diff
@@ -44,3 +59,23 @@ pub fn task<
     let handle = spawn_handle.spawn_essential("aleph/consensus_session_member", task);
     Task::new(handle, stop)
 }
+
+fn rotate_saved_unit_files(stash_path: &Path, session_id: u32) -> Result<(File, File), io::Error> {
+    let extension = ".abfst";
```

Comment:
what is the expansion? :)

Reply:
AlephBFT…state…something, I think

I'm realizing that I should have noted the rationales for these things in the commit message. 😂

Reply:
I would put a short explanation here in code and that should be enough :)

Reply:
May I suggest `.abfts"? 😄

Reply:
Not `units`, since it will contain more staate in the future (at least alert data, if/when we add randomness also keys related to that).

Reply:
Changed to `.abfts`